indico/modules/events/abstracts/fields.py

```diff
@@ -156,8 +156,11 @@ def _get_person_link(self, data):
                       'is_speaker': data.pop('isSpeaker', self.default_is_speaker)}
         return super(AbstractPersonLinkListField, self)._get_person_link(data, extra_data)
 
-    def _serialize_person_link(self, principal, extra_data=None):
-        return serialize_abstract_person_link(principal)
+    def _serialize_person_link(self, principal, extra_data={}):
```

Comment:
can we please not use mutable default arguments? while it doesn't cause problems here it's just risky since someone may accidentally modify it. `extra_data or {}` in the next line will work fine with the `None` default